Dockerfile

```diff
@@ -0,0 +1,19 @@
+# pull official base image
+FROM node:13.12.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
+COPY package-lock.json ./
+RUN npm install --silent
+RUN npm install react-scripts@3.4.0 -g --silent
+
+# add app
+COPY . ./
+# start app
+CMD ["npm", "start"]
```

Comment:
Docker distinguishes between `ENTRYPOINT` and `CMD`.
In order to make your container nice and flexible, you should use that.
--> entrypoint should be npm.
--> cmd should be the npm command you want to run.